Doppler.HtmlEditorApi/Controllers/CampaignsController.cs

```diff
@@ -133,27 +133,25 @@ public async Task<IActionResult> SaveCampaign(string accountName, int campaignId
                     HtmlContent: content,
                     HtmlHead: head,
                     Meta: campaignContent.meta.ToString(),
-                    CampaignId: campaignId,
                     PreviewImage: campaignContent.previewImage),
                 ContentType.html => new HtmlContentData(
                     HtmlContent: content,
                     HtmlHead: head,
-                    CampaignId: campaignId,
                     PreviewImage: campaignContent.previewImage),
                 _ => throw new NotImplementedException($"Unsupported campaign content type {campaignContent.type:G}")
             };
 
             if (campaignState.ContentExists)
             {
-                await _campaignContentRepository.UpdateCampaignContent(accountName, contentRow);
+                await _campaignContentRepository.UpdateCampaignContent(campaignId, contentRow);
```

Comment:
Este cambio de quitar el `accountName` cómo parámetro de `UpdateCampaignContent` parece fuera de scope. 

Tal vez sea correcto, pero es una problema diferente derefactorizar `BaseHtmlContentData`, en todo caso merece un commit independiente.

Reply:
Done!
Related to https://github.com/FromDoppler/doppler-html-editor-api/pull/116#discussion_r878046511